### ISSUE TITLE
npm run dev issue with v.23.2.4

### ISSUE THREAD

**Issue by frankverhorst** [bettyblocks/cli#118]
Updated the CLI to the latest version (v.23.2.4) and noticed that the command `npm run dev` does not work anymore. It is giving this error:

```
[nodemon] 1.19.4
[nodemon] to restart at any time, enter 'rs'
[nodemon] watching dir(s): src/**/*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting 'rimraf dist && bb components build && bb components serve'
An error has occurred, please check if something went wrong during the build step.
[nodemon] clean exit - waiting for changes before restart
```

Temporary reverting back to earlier version.

**Comment by nulian** [comment]
What for message does npm run build give? 
Because it would mean that the dist folder is empty to get that error.

**Comment by frankverhorst** [comment]
Haven't test that. But the issue seems to be resolved in 23.2.6. Thanks!
